lib/Dialect/AIG/Transforms/MaximumAndCover.cpp

```diff
@@ -0,0 +1,92 @@
+//===- Balancing.cpp -AIG Binary Ops depth balancing ------*- C++ -*-===//
```

Comment:
nit: 
```suggestion
//===----------------------------------------------------------------------===//
```

FYI https://github.com/llvm/llvm-project/pull/118553